### DISCUSSION TITLE
Is it possible to rename the top-level directory "packages" to something else?

### DISCUSSION THREAD

**Discussion by maufarinelli** [vercel/turborepo#3369] [Help] [answered]
I am implementing that to our existing repo.
All our common shared internal packages are inside "packages" folder and it works.

As we are sharing our repo with some other code (written in other languages like C Sharp), our .gitignore have `./packages`.
So, I was trying to rename "packages" to something else, but it does not work. 

Is there a way to rename "packages" to something else? 
Some config probably?

**Comment by petermekhaeil** [comment, marked as answer]
It's configured in your package manager. See https://turbo.build/repo/docs/handbook/workspaces#configuring-workspaces

**Comment by maufarinelli** [comment]
Thanks for replying @petermekhaeil 
I changed in my package.json
```
"workspaces": [
    "webapps/*",
    "webpackages/*"
  ],
```
but still does not work

BTW, I am using `yarn`

**Reply by maufarinelli** [reply]
Nevermind, It is working now. 
I think we had some cache preventing
